app/templates/components/github-issue-select.hbs

```diff
@@ -0,0 +1,11 @@
+{{#power-select
+  disabled=task.githubId
+  loadingMessage='Retrieving issues...'
+  options=issues
+  onchange=onIssueSelected
+  placeholder='Connect your task with a GitHub issue'
+  renderInPlace=true
```

Comment:
Not all of them are. The one for task assignment is causing layout problems if we render in place, due to parent scroll bars. 

There is, however, a fix listed in the power-select changelog that may have resolved it. We could look into it, but probably not as part of this issue.